Pipfile

```diff
@@ -9,7 +9,7 @@ verify_ssl = true
 "boto3" = "*"
 onelogin = "*"
 keyring = "*"
-ipify = "*"
+requests = "*"
 
 [requires]
-python_version = ">=3.5"
+python_version = "3.8"
```

Comment:
Does this need to change?  Looks like the changes below will work fine on older versions of python as well.

Reply:
Yes, I think so too--let me see if I can get it working locally without that change and I'll push